foreman/data_refinery_foreman/surveyor/surveyor.py

```diff
@@ -13,7 +13,7 @@ class SourceNotSupportedError(BaseException):
     pass
 
 
-def _get_surveyor_for_source(survey_job: SurveyJob):
+def get_surveyor_for_source(survey_job: SurveyJob):
```

Comment:
I kind of like the convention of having a prefix of `_` to indicate that a function is private. Any reason why you removed it?